### PR TITLE
Fixed the Google icon on Login page

### LINKED ISSUE #72: Enhance the Login Page
**Describe the bug**
REMOVE - 
OR 
Sign in with
The two logos
Shift the whole section at the center of the screen.
Put the sign up section at the bottom.
Change the logo with the new one in the .github/ directory

**Expected behavior**
Login Page UI should be according to the above directions.

### PR DESCRIPTION
**Pull request by akshaydhiran**
loginUIchanges

### CODE REVIEW

**Review by akshaydhiran — COMMENTED** [review]
NewUpdatedLoginScreen

**Review by TahaAli07 — COMMENTED** [review]
@akshaysingh5 We can delete the Facebook and Google icon files now.

**Comment by akshaydhiran** [comment]
Okay

**Review by ankitshaw — CHANGES_REQUESTED** [review]
I feel the logo looks too small and the gap between login button and signup line is large.

**Comment by akshaydhiran** [comment]
@ankitshaw  Okay, I will make the changes now and update you on slack first.

**Review by akshaydhiran — COMMENTED** [review]
LastUpdatedLoginScreen

&


LoginScreen

File: master/lib/screens/login.dart [hunk 1]
```diff
@@ -272,14 +206,14 @@ class _LoginPageState extends State<LoginPage> {
...
-        padding: EdgeInsets.all(25),
+        padding: EdgeInsets.all(20),
         alignment: Alignment.center,
         decoration: BoxDecoration(
           gradient: LinearGradient(colors: [Colors.redAccent,Colors.pink],begin: Alignment.topLeft,end: Alignment.bottomRight),
         ),
         child: Column(
         children: <Widget>[
-          SizedBox(height: 10),
+          SizedBox(height: 80),
```
Comment (ankitshaw) [inline master/lib/screens/login.dart:216]: need for this change?

**Comment by akshaydhiran** [comment]
1st Image is for when we have vertical: 10 on line no. 180 ( Login.dart ).
2nd images is for when we have vertical: 100 on line no. 180 ( Login.dart ).

**Review by ankitshaw — APPROVED** [review]

**Comment by ankitshaw** [comment]
@all-contributors bot please add @akshaysingh5  for fix

**Comment by ankitshaw** [comment]
@all-contributors bot please add @akshaysingh5  for code

**Pull request merged**